tools/esdlcmd/esdl-publish.cpp

```diff
@@ -38,11 +38,12 @@ class EsdlPublishCmdCommon : public EsdlCmdCommon
     StringAttr optTargetAddress;
     StringAttr optTargetPort;
     bool       optOverWrite;
+    bool       optHttps;
```

Comment:
New member variables should be initialized inline, rather than at the constructor (unless the constructor has to logically or otherwise calculate or decide the value.

bool optHttps = false;

Reply:
Even though I eventually want to change it across the board to use --tls, to be consistent with other cammands, the option should be --ssl, and the variable optSSL.

Reply:
In ecl command line code the option is actually in the base class EclCmdCommon.

Reply:
Should --ssl also go in the base class for the esdl tool?

Reply:
@asselitx not exactly sure what you mean by the base class for the esdl tool.  EsdlCMDShell?

Reply:
@afishbeck I meant in EsdlCmdCommon. Wasn't sure if you were implying I should follow the pattern set by the ecl command line tool.